src/player_utils.c

```diff
@@ -723,7 +723,7 @@ void init_player_as_single_keeper(struct PlayerInfo *player)
     struct InitLight ilght;
     memset(&ilght, 0, sizeof(struct InitLight));
     player->field_4CD = 0;
-    ilght.radius = game.conf.rules.game.hand_light_radius;
+    ilght.radius = game.conf.rules.game.hand_light_radius * COORD_PER_STL;
```

Comment:
You don't want to do it where we configure it like here?
https://github.com/dkfans/keeperfx/commit/b36b3ef39b74d08d57f28b2c7719347a115dccac#diff-1bf5b73865ef053252f6a94089db492d69cd359a860427bbcd544209df9166d7R190